js/apps/admin-ui/src/clients/advanced/AdvancedSettings.tsx

```diff
@@ -160,6 +160,35 @@ export const AdvancedSettings = ({
               )}
             />
           </FormGroup>
+          <FormGroup
```

Comment:
Should we be hiding this field based on whether the `DPOP` profile is enabled? We have this information in the environment variable injected to the front-end.

Reply:
Yes, we should hide the field about DPoP if DPoP profile is disabled. I will investigate into how to do it.